src/infrastructure/api/event-router/event-routes.ts

```diff
@@ -13,6 +13,8 @@ eventRouter.post('/events/newMessage', EventController.addMessageEvent);
 
 eventRouter.post('/events/newParticipant', addParticipantDto, EventController.addParticipant);
 
+eventRouter.post('/events/newAssistent', addAssistentDto, EventController.addAssistent);
```

Comment:
```suggestion
eventRouter.post('/events/newAssistent', addAssistentDto, checkInUseCase);
```

La ruta ha de portar al use case de fer check in, I do el use case crida al controlador per afegir l'assistent